client/src/components/RestaurantShow.js

```diff
@@ -4,20 +4,17 @@ import { Link } from "react-router-dom"
 import ReviewList from "./ReviewList.js"
 import NewReviewForm from "./NewReviewForm.js"
 import ErrorList from "./layout/ErrorList.js";
+import translateServerErrors from "../services/translateServerErrors.js";
 
 const RestaurantShow = ({ currentUser, ...props }) => {
-  const emptyRestaurant = {
-    reviews: []
-  }
-  const [restaurant, setRestaurant] = useState(emptyRestaurant)
+  const [restaurant, setRestaurant] = useState({})
   const [reviewListOrForm, setReviewListOrForm] = useState("list")
   const [errors, setErrors] = useState({})
   const { id } = props.match.params
-  
-  const getRestaurant = async () => {
 
+  const getRestaurant = async () => {
     try {
-      const response = await fetch(`/api/v1/restaurants/${id}`)
+      const response = await fetch(`/api/v1/restaurants/${id}/${currentUser?.id}`)
```

Comment:
wooohhh. Not sure why we are adding in the user id here? regardless, this is both not needed and not ideal

you can always see the current user on the backend by called `req.user`